src/clj/witan/send/send.clj

```diff
@@ -309,10 +323,11 @@
         transitions (if (nil? modified-transition-matrix)
                       (u/transitions-map transition-matrix)
                       (u/transitions-map modified-transition-matrix))
-        transition-matrix-filtered (if (nil? modified-transition-matrix)
-                                     (filter #(= (:calendar-year %) 2016) transition-matrix)
-                                     (filter #(= (:calendar-year %) 2016) modified-transition-matrix))
-
+        transition-matrix-filtered (if (nil? filter-transitions-from)
+                                     nil
+                                     (if (nil? modified-transition-matrix)
+                                       (filter #(= (:calendar-year %) filter-transitions-from) transition-matrix)
```

Comment:
Is `filter-transitions-from` a single year, or a list of years? If it's just a single year, won't this only give only one year of data in `transitions-matrix-filtered`?

Reply:
yes you're right! Previously we've only wanted to filter from final year so it's only ever been one year, but it would make more sense to allow multiple years. good catch!